meson.build

```diff
@@ -249,8 +249,13 @@ gcab = find_program('gcab', required : get_option('tests'))
 bashcomp = dependency('bash-completion', required: false)
 python3 = find_program('python3')
 
-if get_option('plugin_uefi_pk')
+if get_option('gnutls')
```

Comment:
As it's not at all obvious that the synaptics rmi plugin gets gimped without gnutls support, I think we need to make noise if this is turned off.  Perhaps we rename `plugin_synaptics` to `plugin_synaptics_mst` and then add another `plugin_synaptics_rmi` that can emit a warning?

Reply:
https://github.com/fwupd/fwupd/pull/2753